Cargo.toml

```
@@ -40,10 +42,33 @@ required-features = ["http-client"]
 name = "reply_to_message_updates"
 required-features = ["http-client"]
 
+[dependencies.async-trait]
+version = "0.1"
+optional = true
+
+[dependencies.mime_guess]
+version = "2"
+optional = true
+
+[dependencies.multipart]
+version = "0.18"
+default-features = false
+features = ["client"]
+optional = true
+
 [[example]]
 name = "reply_keyboard"
 required-features = ["http-client"]
 
+[dependencies.tokio]
```

I think we can actually get fully rid of the second feature:

```suggestion
[target.'cfg(not(target_arch = "wasm32"))'.dependencies.tokio]
```

`async-http-client=[…, "dep:tokio"]` then ignores `tokio` when it's not configured for a given `target_arch` but includes it when it's there.